File: java/disk.java
public class disk {
    // atribut
    private String type;
    private int capacity;  
    private int price;
    
    // constructor
    disk(){}

    disk(String type, int capacity, int price){
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    // setter dan getter

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }


}

File: latihan3/cpp/disk.cpp
class disk{

    private:
        // atribut
        string type;
        int capacity;
        int price;

    public:
        // konstruktor
        disk(){}

        disk(string type, int capacity, int price){
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }

        // setter dan getter
        void setTypeD(string type){ this->type = type; }
        string getTypeD(){ return this->type; }

        void setCapacityD(int capacity){ this->capacity = capacity; }
        int getCapacityD(){ return this->capacity; }

        void setPriceD(int price){ this->price = price; }
        int getPriceD(){ return this->price; }

        // destructor
        ~disk(){}
};